src/components/toast/readme.md

```diff
@@ -39,7 +39,7 @@ A `sbb-toast` can also be given a custom action that, if marked with the `sbb-to
   <!-- Toast action can be a button -->
   <sbb-button slot="action" icon-name="clock-small" sbb-toast-close></sbb-button>
   <!-- Or a link -->
-  <sbb-link slot="action">Link action</sbb-link>
+  <sbb-link-button slot="action">Link action</sbb-link-button>
```

Comment:
Since only the inline version is supported, I would like to add a note in the readme to specify this to the consumers.